app/models/assessment.rb

```diff
@@ -20,7 +20,7 @@ class Assessment < ApplicationRecord
   has_one :scoreboard, dependent: :destroy
 
   # Validations
-  validates :name, uniqueness: { scope: :course_id }
+  validates :name, uniqueness: { case_sensitive: true, scope: :course_id }
```

Comment:
I think it should not be case_sensitive. It be weird to have two assessments "apple pie" and "Apple Pie"